Signum.Engine/Operations/OperationLogic.cs

```diff
@@ -142,15 +142,23 @@ public static void RegisterPreviousLog<T>()
 
         public static void ExceptionLogic_DeleteLogs(DeleteLogParametersEmbedded parameters, StringBuilder sb, CancellationToken token)
         {
+            void Remove(DateTime dateLimit, bool withExceptions)
+            {
+                var query = Database.Query<OperationLogEntity>().Where(o => o.Start < dateLimit);
+
+                if (withExceptions)
+                    query.Where(a => a.Exception != null).UnsafeDeleteChunksLog(parameters, sb, token);
+                else
+                    query.Where(a => a.Exception == null).UnsafeDeleteChunksLog(parameters, sb, token);
+            }
+

```

Comment:
There are many modules using GetDateLimitDelete / GetDateLimitDeleteWithExceptions and all of them are considering GetDateLimitDate as Where(true), not Where(Exception == null). 

If you think is counter intuitive you need to change them all, but for me it's ok. 

Also, I consider a code smell to make two calls to a method that then has an if splitting for each call site. I would just avoid the method or use a `Expression<Func<OperationLog, bool>>` as a parameter to avodi the if(withExceptions)